src/Cronofy/UserInfo.cs

```diff
@@ -225,85 +232,189 @@ public override string ToString()
-            public string DelegatedScope { get; set; }
+          /// <summary>
+          /// Gets or sets the name of the provider of the profile.
+          /// </summary>
+          /// <value>
+          /// The name of the provider for the profile.
+          /// </value>
+          public string ProviderName { get; set; }
+
+          /// <summary>
+          /// Gets or sets the ID of the profile.
+          /// </summary>
+          /// <value>
+          /// The ID of the profile.
+          /// </value>
+          public string Id { get; set; }
+
+          /// <summary>
+          /// Gets or sets the name of the profile.
+          /// </summary>
+          /// <value>
+          /// The name of the profile.
+          /// </value>
+          public string Name { get; set; }
+
+          /// <summary>
+          /// Gets or sets a value indicating whether this profile is connected.
+          /// </summary>
+          /// <value>
+          /// <c>true</c> if this profile is connected; otherwise, <c>false</c>.
+          /// </value>
+          public bool Connected { get; set; }
+
+          /// <summary>
+          /// Gets or sets the relink URL for the profile.
+          /// </summary>
+          /// <value>
+          /// The relink URL for the profile.
+          /// </value>
+          /// <remarks>
+          /// <c>null</c> unless <see cref="Connected"/> is false.
+          /// </remarks>
+          public string RelinkUrl { get; set; }
+
+          /// <inheritdoc/>
+          public override int GetHashCode()
+          {
+            return this.Id.GetHashCode();
+          }
+
+          /// <inheritdoc/>
+          public override bool Equals(object obj)
+          {
+            var other = obj as ConferencingProfile;
 
-            /// <inheritdoc/>
-            public override int GetHashCode()
+            if (other == null)
             {
-                return this.Scope.GetHashCode();
+              return false;
             }
 
-            /// <inheritdoc/>
-            public override bool Equals(object obj)
-            {
-                var other = obj as Authorization;
+            return this.Equals(other);
+          }
+
+          /// <summary>
+          /// Determines whether the specified <see cref="Cronofy.UserInfo.ConferencingProfile"/> is
+          /// equal to the current <see cref="Cronofy.UserInfo.ConferencingProfile"/>.
+          /// </summary>
+          /// <param name="other">
+          /// The <see cref="Cronofy.UserInfo.ConferencingProfile"/> to compare with the current
+          /// <see cref="Cronofy.UserInfo.ConferencingProfile"/>.
+          /// </param>
+          /// <returns>
+          /// <c>true</c> if the specified <see cref="Cronofy.UserInfo.ConferencingProfile"/> is equal
+          /// to the current <see cref="Cronofy.UserInfo.ConferencingProfile"/>; otherwise,
+          /// <c>false</c>.
+          /// </returns>
+          public bool Equals(Profile other)
```

Comment:
```suggestion
          public bool Equals(ConferencingProfile other)
```

I believe theres a possible infinite loop  when calling `Equals(object obj)` as this parameter needs to be the type of the class itself.

Reply:
Great pick up.